app/config.js

```diff
@@ -9,6 +9,6 @@ angular.module("ngApp.config", [])
     "auth_api": "__AUTH__/api",
     "supportedAPI": "1",
     "tag": "https://github.com/NCI-GDC/portal-ui/releases/tag/__VERSION__",
-    "production": __PRODUCTION__,
-    "fake_auth": __FAKE_AUTH__,
+    "production": "__PRODUCTION__",
+    "fake_auth": "__FAKE_AUTH__",
```

Comment:
tests threw `__PRODUCTION__ undefined` now that it's not loading ngApp though might not be an issue anymore. I'll do some cleanups in the morning.